packages/costa/src/runtime.ts

```diff
@@ -18,7 +19,21 @@ import Debug from 'debug';
 const debug = Debug('costa.runtime');
 const CONDA_CONFIG = path.join(__dirname, '../node_modules/@pipcook/boa/.CONDA_INSTALL_DIR');
 
-function selectLatestPackage(metadata: NpmPackageMetadata): NpmPackage {
+function selectNpmPackage(metadata: NpmPackageMetadata, source: PluginSource): NpmPackage {
+  const { version } = source?.schema;
+  if (version === 'beta') {
+    if (metadata['dist-tags'].beta == null) {
```

Comment:
=== or just !metadata['dist-tags'].beta ?

Reply:
`==` with null accept the value `undefined` and `null`.